src/components/projectDetail/AddressAddModal.tsx

```
@@ -0,0 +1,110 @@
+              className="px-4 py-3 rounded-lg"
+              onClick={() => setIsAddressModalOpen(true)}
+            >
+              찾기
+            </SecondaryButton>
+          </div>
+        </div>
+        <div>
+          <label className="block text-sm font-medium text-sub-gray mb-2">상세 주소</label>
+          <input
+            type="text"
+            className="w-full p-3 border border-gray-border rounded-lg"
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)